backend/src/main/java/com/bakdata/conquery/models/worker/Namespace.java

```diff
@@ -59,17 +60,17 @@
 
 	private final FilterSearch filterSearch;
 
-	public Namespace(DatasetRegistry datasetRegistry, NamespaceStorage storage, boolean failOnError, ObjectWriter objectWriter, CSVConfig csvConfig) {
+	public Namespace(DatasetRegistry datasetRegistry, NamespaceStorage storage, boolean failOnError, ObjectWriter objectWriter, CSVConfig csvConfig, SearchConfig searchConfig) {
 		namespaces = datasetRegistry;
 		this.storage = storage;
 		executionManager = new ExecutionManager(this);
 		jobManager = new JobManager(storage.getDataset().getName(), failOnError);
 
-		filterSearch = new FilterSearch(storage, jobManager, csvConfig);
+		filterSearch = new FilterSearch(storage, jobManager, csvConfig, searchConfig);
 
 		this.objectWriter = objectWriter;
 
-		namespaces.add(this);
+		datasetRegistry.add(this);
```

Comment:
Kannst du `namespaces` bitte noch aus der Klasse entfernen? Das wird soweit ich sehen kann nur in den Tests benutzt und dann kann hier auch die Zuweisung raus